spec/daru/io/importers/xlsx_spec.rb

```diff
@@ -0,0 +1,67 @@
+RSpec.describe Daru::IO::Importers::XLSX do
+  subject { described_class.new(path, sheet: sheet, headers: headers).call }
+
+  let(:sheet)   { 0    }
+  let(:headers) { true }
+
+  context 'parses default sheet' do
```

Comment:
Yes, it is logical. What I am concerned about here is by concept description, you are testing "what it would be if there's no sheet param passed" (I am guessing it by "default" word). But in fact, you are testing the `sheet: 0` explicitly passed.